### PR TITLE
Add items counter

### LINKED ISSUE #3: [3pt]  - Add tests for items counter - Student A
Make sure that function for the items counter is covered with unit tests using Jest. Make sure that you have tests for all edge cases.

### LINKED ISSUE #6: [3pt]  - Add all items counter on the Homepage - Student A
Even if the API gives you the number of items, you will create a specific function to calculate it.

Make sure that the correct number is displayed on the Homepage.

Respect the following rules:

- Each counter should be implemented as a separate module.
- A counter function should look for specific DOM elements (e.g. for the comments counter it should look for comments) and make the counting based on what is actually displayed on the page.
- A counter function should cover all the edge cases you can think about.

### PR DESCRIPTION
**Pull request by wassimchakib**
In this feature, I:

- Added item counter function to count the number of games shown on the page
- Added test for the function to cover edge cases

### CODE REVIEW

**Review by manq2010 — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @wassimchakib , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-  No linter errors. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark:
-  Test cases passed. :heavy_check_mark: 



## Optional suggestions

- See optional suggestions on the comments below

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: tests/itemCounter.test.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    const result = countLists(cards);
+
+    expect(result).toBe(0);
+  });
+
+  test('countLists returns 1', () => {
+    const cards = document.querySelector('.cards');
+    const paragraph = document.createElement('p');
+    cards.appendChild(paragraph);
```
Comment (manq2010) [inline tests/itemCounter.test.js:33]: [OPTIONAL] Good job with the test, however, I think it would be better if you call the function that adds children to the parent container and check if the parent was appended.

File: src/modules/dom.js [hunk 2]
```diff
@@ -72,6 +73,11 @@ const updateDOM = () => {
         });
       });
 
+      // Count number of games
+      const navGame = document.querySelector('.nav-games');
+      const numberOfDisplayedGames = countListOfGames(cards);
+      navGame.textContent = `Games (${numberOfDisplayedGames})`;
+
```
Comment (manq2010) [inline src/modules/dom.js:80]: [OPTIONAL] Good job with the lines of code, however, I think it's better to have it as a function or separate module so it can be reusable to determine child elements for any parent. This will ensure your code is not DRY
